text/0017-coretime-market-redesign.md

```diff
@@ -0,0 +1,134 @@
+
+All current tenants have 7 days to decide whether they want to renew their core or not. After obtaining the information who renewed and who did not, the system has the necessary information to conclusively allocate all cores and transfer ownership.
+
+In the case where there are combined more renewals and bidders at or above the `CLEARING_PRICE` than available cores, we allocate cores to the highest to lowest bidders until all available cores are allocated (albeit still at the `CLEARING_PRICE`). That effectively means that in situations with very high demand, some bidders might not get the coretime they bid for.
+
+If the supply exceeds the demand, all unallocated cores are transferred to the Instantanous Market.
+
+#### Reserve Price Adjustment
+
+After all cores are allocated, the `RESERVE_PRICE` is adjusted similar to the process described in RFC-1, where we define `TARGET_CONSUMPTION_RATE` as a ratio of the available to unsold cores. Then, the upcoming reserve price is adjusted upwards or downward, depending on whether we over- or undershoot the target consumption. If the consumption is met precisely, the price remains unchanged in the next `BULK_PERIOD`. 
+
+**Note**: When designing this mechanism, we want to make sure that small deviations have a smaller price impact than bigger deviations. We propose the following function:
+
+`reserve_price_old`: reserve price from the previous period
+`consumption_rate`: how many cores were sold relative to how many were available.
+`TARGET_CONSUMPTION_RATE`: how many of the available cores we want to sell without increasing the price. We propose 90%. This leaves enough area downward and upward to adjust prices more aggressively.
+`K`: sensitivity parameter. How aggressively we want to adjust the price. We propose a value between 2 and 3, but might need more evaluation.
+`P_MIN`: A minimum price we never undercut. This is important to bound the price downward and prevent computational issues if prices drop too low. We propose a value of 1 DOT.
+`MIN_INCREMENT`: A minimum increment after we reached 100% capacity. This is important to quickly recover after long periods of low demand which resulted in low prices.
+
+We update the price according to:
+
+`p_new = reserve_price_old * exp(K * (consumption_rate - TARGET_CONSUMPTION_RATE))`
+
+The `RESERVE_PRICE` in the next period will be:
+
+`RESERVE_PRICE_NEXT = max(p_new, P_MIN)`
+
+**Note:** To reduce the recovery time from very low prices it is important to, in the case of 100% capacity, at least increment the `RESERVE_PRICE_NEXT` by `MIN_INCREMENT`, which could be, e.g., 100 DOT.
```

Comment:
I'm not convinced about this `MIN_INCREMENT` mechanism being a step change, but after conversations in person I know you're not a fan of my alternative (some function which defines a point below which corrections are increasingly large the further from the point it gets). This point would then be set by governance, rather than setting the step as here.

Also it's probably obvious to many readers but worth making it explicit that this model is proposed under the assumption that the `MIN_INCREMENT` parameter is exposed to governance to set and change as time goes on. The same goes for other values such as `TARGET_CONSUMPTION_RATE`, `K`, `P_MIN`